src/components/tl/nccl/tl_nccl_coll.c

```diff
@@ -101,7 +112,7 @@ ucc_status_t ucc_nccl_alltoall_start(ucc_coll_task_t *coll_task)
     return status;
 }
 
-ucc_status_t ucc_tl_nccl_alltoall_init(ucc_tl_nccl_task_t *task)
+ucc_status_t ucc_nccl_alltoall_init(ucc_tl_nccl_task_t *task)
```

Comment:
It seems UCP team followed "`ucc_tl_<team>_xx`". any reason for this change?

Reply:
I can rename it back to ucc_tl_, but shorter function names when there is no confusion looks better I think.

Reply:
IMHO, either add _tl here (or) remove _tl from ucp func . @vspetrov  WDYT?

Reply:
@Sergei-Lebedev agree with @bureddy the main thing to be consistent. I personally would just keep the way it is: ucc_tl_ ucc_cl_, etc.

Reply:
fixed